internal/sdkprovider/service/organization/organization_user_group_member.go

```diff
@@ -0,0 +1,126 @@
+package organization
+
+import (
+	"context"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+
+	"github.com/aiven/aiven-go-client"
+	"github.com/aiven/terraform-provider-aiven/internal/schemautil"
+	"github.com/aiven/terraform-provider-aiven/internal/schemautil/userconfig"
+)
+
+var aivenOrganizationUserGroupMemberSchema = map[string]*schema.Schema{
+	"organization_id": {
```

Comment:
Isn't the organization_id given by the group?

Reply:
it is, why?